lib/mining/miner.js

```diff
@@ -235,7 +236,7 @@ class Miner extends EventEmitter {
   getAddress() {
     if (this.addresses.length === 0)
       return new Address();
-    return this.addresses[Math.random() * this.addresses.length | 0];
+    return this.addresses[random.randomRange(0, this.addresses.length)];
```

Comment:
Checked by adding 3 addresses to new `Miner` and calling this method repeatedly. All 3 addresses returned in roughly equal distribution, no errors.